vanilla-framework/vanilla-docs-theme
# Added extra styles for --separate element. Updated padding and margins

## Done

Reduced aside padding and margins.
Added `.p-toc__item` style and `--separate` modifier (for styling visually separated elements).
Added `--active` modifier.

## QA

1. Pull this branch
2. Run `./run watch`
3. Save https://gist.github.com/Lukewh/dc0f1b9d20a9ef46bb337fac43c03502 to a html file in the parent directory of this repo.
4. Open the saved file and see the toc on the right hand side
5. Scroll up and down and resize the browser

## Details

Fixes: https://github.com/vanilla-framework/vanilla-docs-theme/issues/1.
Needed for new fixed position niceness: https://github.com/ubuntudesign/docs.vanillaframework.io/pull/106

## Screenshots

<img width="229" alt="screen shot 2017-07-18 at 12 08 02" src="https://user-images.githubusercontent.com/479384/28314279-cc5fc8fa-6bb1-11e7-8193-f52dfc2b624c.png">
<img width="228" alt="screen shot 2017-07-18 at 12 08 08" src="https://user-images.githubusercontent.com/479384/28314282-cdbf6d4a-6bb1-11e7-83ba-1656bd2e0630.png">
